backend/CircleForms/Contracts/ContractModels/Response/AnswerContract.cs

```diff
@@ -1,14 +1,14 @@
-﻿using System.Collections.Generic;
-using CircleForms.Models.Posts.Questions.Submissions;
-using Newtonsoft.Json;
-
-namespace CircleForms.Contracts.ContractModels.Response;
-
-public class AnswerContract
-{
-    [JsonProperty("submissions")]
-    public List<Submission> Submissions { get; set; }
-
-    [JsonProperty("user")]
-    public UserAnswerContract User { get; set; }
-}
+﻿using System.Collections.Generic;
+using CircleForms.Database.Models.Posts.Questions.Submissions;
+using Newtonsoft.Json;
+
+namespace CircleForms.Contracts.ContractModels.Response;
+
+public class AnswerContract
+{
+    [JsonProperty("submissions")]
+    public List<Submission> Submissions { get; set; }
+
+    [JsonProperty("user")]
+    public string UserId { get; set; }

```

Comment:
Wouldn't it be better to have this as int or long?

Reply:
Our ids are strings